### PR TITLE
docs: changelog for 5.11.2

### PR DESCRIPTION
**Pull request by PeachScript**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

- 🆕 Table with `virtual` can now customize `components` except the `components.body`. [#45857](https://github.com/ant-design/ant-design/pull/45857)
- 🆕 Add Menu properties `popupBg` and `darkPopupBg` to customize popup background. [#45826](https://github.com/ant-design/ant-design/pull/45826)
- 🐞 Fix Button with href and disabled that could be focused. [#45910](https://github.com/ant-design/ant-design/pull/45910) [@MadCcc](https://github.com/MadCcc)
- 🐞 Fix `zIndex` logic problem that message and notification are covered when multiple Modal are opened. [#45911](https://github.com/ant-design/ant-design/pull/45911) [#45864](https://github.com/ant-design/ant-design/pull/45864) [@kiner-tang ](https://github.com/kiner-tang)
- 💄 Fix QRCode `style.padding` is not working. [#45815](https://github.com/ant-design/ant-design/pull/45815)
- 💄 Optimize Carousel dots border radius style. [#45817](https://github.com/ant-design/ant-design/pull/45817)
- TypeScript
  - 🤖 Optimize List `gutter` property type definition. [#45791](https://github.com/ant-design/ant-design/pull/45791) [@Asanio06](https://github.com/Asanio06)

---

- 🆕 放开 Table `virtual` 下 `components` 的限制，现在除了 `components.body` 都可以自定义。[#45857](https://github.com/ant-design/ant-design/pull/45857)
- 🆕 新增 Menu 属性 `popupBg` 及 `darkPopupBg` 以自定义浮层背景色。[#45826](https://github.com/ant-design/ant-design/pull/45826)
- 🐞 修复 Button 带有链接且禁用时可以被聚焦到的问题。[#45910](https://github.com/ant-design/ant-design/pull/45910) [@MadCcc](https://github.com/MadCcc)
- 🐞 修复 `zIndex` 逻辑，解决多层 Modal 打开时，message 与 notification 被遮盖的问题。[#45911](https://github.com/ant-design/ant-design/pull/45911) [#45864](https://github.com/ant-design/ant-design/pull/45864) [@kiner-tang ](https://github.com/kiner-tang)
- 💄 修复 QRCode 设置 `style.padding` 时无效的问题。[#45815](https://github.com/ant-design/ant-design/pull/45815)
- 💄 优化 Carousel 切换条圆角样式。[#45817](https://github.com/ant-design/ant-design/pull/45817)
- TypeScript
  - 🤖 优化 List 属性 `gutter` 的类型定义。[#45791](https://github.com/ant-design/ant-design/pull/45791) [@Asanio06](https://github.com/Asanio06)

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 1b6297c</samp>

This pull request updates the documentation and the package.json file for the release of version 5.11.2 of `antd`. It adds the release notes in English and Chinese and changes the version number in the `package.json` file.

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 1b6297c</samp>

* Bump version number to 5.11.2 in `package.json` ([link](https://github.com/ant-design/ant-design/pull/45946/files?diff=unified&w=0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L3-R3))
* Add release notes for 5.11.2 in English and Chinese in `CHANGELOG.en-US.md` and `CHANGELOG.zh-CN.md` ([link](https://github.com/ant-design/ant-design/pull/45946/files?diff=unified&w=0#diff-e961ad288c43a2feb2d56b696e10c0b670d652dbb17c76c9aa0452d45157977cR19-R31), [link](https://github.com/ant-design/ant-design/pull/45946/files?diff=unified&w=0#diff-1224d5ca5233fdeba67ba2e2dc6fa3e615905253f35dc132c4b4b86f51f9eab4R19-R31))

### CODE REVIEW

**Review by kiner-tang — APPROVED** [review]

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -16,6 +16,19 @@ tag: vVERSION
 
 ---
 
+## 5.11.2
+
+`2023-11-17`
+
+- 🆕 放开 Table `virtual` 下 `components` 的限制，现在除了 `components.body` 都可以自定义。[#45857](https://github.com/ant-design/ant-design/pull/45857)
+- 🆕 新增 Menu 属性 `popupBg` 及 `darkPopupBg` 以自定义浮层背景色。[#45826](https://github.com/ant-design/ant-design/pull/45826)
```
Comment (li-jia-nan) [inline CHANGELOG.zh-CN.md:24]: 这个是不是又回滚了？

Comment (PeachScript) [reply CHANGELOG.zh-CN.md:24]: 👍好像是的，但看起来回滚行数和 PR 行数有区别， @MadCcc 帮忙确认下

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -16,6 +16,19 @@ tag: vVERSION
 
 ---
 
+## 5.11.2
+
+`2023-11-17`
+
+- 🆕 Table with `virtual` can now customize `components` except the `components.body`. [#45857](https://github.com/ant-design/ant-design/pull/45857)
+- 🆕 Add Menu properties `popupBg` and `darkPopupBg` to customize popup background. [#45826](https://github.com/ant-design/ant-design/pull/45826)
```
Comment (MadCcc) [inline CHANGELOG.en-US.md:24]: 这条 revert 了

File: CHANGELOG.zh-CN.md (hunk 1 above)
Comment (PeachScript) [reply CHANGELOG.zh-CN.md]: 已删除

File: CHANGELOG.en-US.md (hunk 2 above)
Comment (PeachScript) [reply CHANGELOG.en-US.md]: 已删除

**Pull request merged**